pkg/params/params.go

```diff
@@ -121,6 +123,12 @@ func FromEnvironment(isOpenshift bool) (EnvConfig, error) {
 		res.DeployServiceMonitors = true
 	}
 
+	if os.Getenv("DEPLOY_FRRK8S_FROM_CNO") == "true" {
+		res.MustDeployFRRK8sFromCNO = true
+	}
+
+	res.FRRK8sDefaultNamespace = os.Getenv("FRRK8S_DEFAULT_NAMESPACE")
```

Comment:
not sure I understand the rationale (and the naming - something that is "default" being fed from outside) of this env var when there's the namespace in the crd. mind expanding a bit? also what happens when it is empty as the charts don't allow external + empty ns:
`- --frrk8s-namespace={{ required "namespace is required when frrk8s is external" .Values.frrk8s.namespace }}`

Reply:
yep we probably need to define the default better